client/src/Header.js

```diff
@@ -1,16 +1,42 @@
 import Logo from "./assets/images/CTY-logo-rectangle.png";
 import { NavLink } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { useState, useEffect } from "react";
 import "./Header.css";
 
 const Header = () => {
 	const [role, setRole] = useState(null);
+	const navigate = useNavigate();
 
 	useEffect(() => {
 		const userType = localStorage.getItem("userType");
 		setRole(userType);
 	}, []);
 
+	const handleLogout = async () => {
+		try {
+			const token = localStorage.getItem("token");
+
+			const response = await fetch("/logout", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+					Authorization: `Bearer ${token}`,
+				},
+			});
+
+			if (response.ok) {
+				localStorage.removeItem("token");
+				localStorage.removeItem("userType");
+				navigate("/");
+			} else {
+				console.error("Logout failed");
+			}
+		} catch (error) {
+			console.error("Error during logout:", error);
+		}
```

Comment:
Don't just log errors. Let the user know that something happened.

Reply:
@adniyaYousaf , make sure you address all my comments before updating a PR. You didn't address this comment

Reply:
@adniyaYousaf , once again, you didn't address this comment. Please make sure you address all comments before updating your PRs

Reply:
I opologise. I misuterstood the comment. I was fixing it api.js.

Reply:
Is that messag alright ? `"Failed to logout. Try again!"`

Reply:
@adniyaYousaf , the problem is not what the message says. The problem is that the user will never see the message because users can't see messages logged to the console.

Show the message to the user somehow. An alert would be the easiest way.